core/src/main/java/org/infinispan/persistence/spi/AdvancedCacheWriter.java

```diff
@@ -35,7 +35,7 @@
     * Callback to be notified when an entry is removed by the {@link #purge(java.util.concurrent.Executor,
     * org.infinispan.persistence.spi.AdvancedCacheWriter.PurgeListener)} method.
     */
-   public interface PurgeListener<K> {
+   interface PurgeListener<K> {
```

Comment:
I thought that we could remove the public modifier as it is redundant for inner interfaces and therefore doesn't change the accessibility of PurgeListener?